lib/ui/semantics.dart

```diff
@@ -604,7 +604,11 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
   /// The field `platformViewId` references the platform view, whose semantics
   /// nodes will be added as children to this node. If a platform view is
   /// specified, `childrenInHitTestOrder` and `childrenInTraversalOrder` must be
-  /// empty, and other semantic configurations will be ignored.
+  /// empty, and other semantic configurations will be ignored. The ignored configurations
+  /// include:[flags], [actions], [textSelectionBase], [textSelectionExtent], [scrollChildren],
+  /// [scrollIndex], [scrollPosition], [scrollExtentMax], [scrollExtentMin], [elevation],
+  /// [thickness], [label], [hint], [value], [increasedValue], [decreasedValue], [textDirection],
```

Comment:
This explicit list seems hard to maintain. Nobody will remember to add a property here if we add a new one.

Reply:
This is a good point, we also have the same problem in the framework semantics code. I think one  problem could be that we have a very long list of properties inside these semantics class, this can lead to maintain hell. 
There are ways I can think of.
1. We can refactor SemanticNode, SemanticConfiguration(in framework), SemanticData(in framework), SemanticProperty(in framework) classes, group this long list of properties to make them shorter. For example, the semanticNode can have `scrollingConfig`, `textConfig`, `actionConfig`, `transformConfig`, etc and each contains reasonable amount of properties. This requires a lot of refactoring work and regression test, and it is also a breaking change. But I feel like it would be a better long term solution since it can be easily scaled up.
2. A non breaking way: We can have a different semantic node class to represent a node for platform view. 

Do you have a different way to do it?  Also copying @cbracken  on this. Maybe we should open a new issue to track this effort?

Reply:
1) would probably be nice, but since it is a breaking change I am not sure if it is currently worth doing. It also doesn't fix the problem. You'd still have to keep a (now shorter) list upto date.

I do not understand what you mean by 2.

Instead of listing all these properties, I'd probably just explain what it means on the platform side to set a paltformViewId.

Reply:
After offline discussion with @goderbauer, I am going to restructure the platform view subtree.